### PR TITLE
license add (CSI-2484)

### PR DESCRIPTION
**Pull request by barakda**

### CODE REVIEW

**Review by ArbelNathan — APPROVED** [review]
LGTM

File: deploy/olm-catalog/ibm-block-csi-operator/1.5.0/manifests/ibm-block-csi-operator.v1.5.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -21,6 +21,9 @@ metadata:
             "name": "ibm-block-csi"
           },
           "spec": {
+            "license": {
+              "accept": "true",
+            },
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator/1.5.0/manifests/ibm-block-csi-operator.v1.5.0.clusterserviceversion.yaml:26]: is it only for red hat? or should it be added to community csv yaml as well?

**Comment by matt-levan** [comment]
I think the idea is that the license defaults to false and the user must manually change it to true in a sort of acknowledging the acceptance of licenses used.

**Comment by barakda** [comment]
A new PR will be opened after getting all the required information (like do we need to add license to community and build a license text/link for this purpose)

**Pull request closed without merging**